src/resources/Activities/tests/Activities.spec.ts

```diff
@@ -36,34 +40,39 @@ describe('Activity', () => {
     describe('list', () => {
         it('should make a POST call to the specific Activity url', () => {
             const params: ListActivitiesParams = {};
+            const activityFacet: ActivityListingFilters = {};
 
-            activity.list(params);
+            activity.list(params, activityFacet);
             expect(api.post).toHaveBeenCalledTimes(1);
-            expect(api.post).toHaveBeenCalledWith(`${Indexes.baseUrl}/public`);
+            expect(api.post).toHaveBeenCalledWith(`${Activity.getBaseUrl()}/public`, activityFacet);
         });
 
         it('should make a POST call to the specific Activity url with the facetsOnly param set as true', () => {
-            const params: ListActivitiesParams = {facetsOnly: true};
+            const params: ListActivitiesFacetsParams = {};
+            const activityFacet: ActivityListingFilters = {};
 
-            activity.list(params);
+            activity.listFacets(params, activityFacet);
```

Comment:
`listFacets` should be tested in a separate describe block, this is the describe for the `list` call

Reply:
gotcha